### DISCUSSION TITLE
New version of open-webui > 0.4 does not support company proxy server for websearching?

### DISCUSSION THREAD

**Discussion by codingstube** [open-webui/open-webui#7340] [Issues]
Hello everyone,
Recently I have the problem that the web requests can no longer be answered and run into a timeout. we “only” have an http_proxy (no https).
I have of course passed the proxy via environment variable into the docker container

thanks

`WARNI [langchain_community.document_loaders.web_base] Error fetching https://de.wikipedia.org/wiki/Nettetal with attempt 2/3: Cannot connect to host de.wikipedia.org:443 ssl:False [Connect call failed ('185.15.59.224', 443)]. Retrying...
WARNI [langchain_community.document_loaders.web_base] Error fetching https://www.bob-automobile.de/duesseldorf with attempt 1/3: Cannot connect to host www.bob-automobile.de:443 ssl:False [Connect call failed ('3.120.181.208', 443)]. Retrying...
WARNI [langchain_community.document_loaders.web_base] Error fetching https://www.ulmen.com/ with attempt 1/3: Cannot connect to host www.ulmen.com:443 ssl:False [Connect call failed ('85.13.149.201', 443)]. Retrying...`

**Comment by ips972** [comment]
+1 - I've opened a similer issue with this:
https://github.com/open-webui/open-webui/discussions/7008

**Comment by codingstube** [comment]
Its a bit weird, the "rag" websites scrapping is working like #https://example.com, but with enabled websearch I get the timeout.
